docs/preview/features/correlation.md

```diff
@@ -108,4 +108,64 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 }
 ```
 
+## Using secret store within Azure Functions
+
+### Installation
+
+For this feature, the following package needs to be installed:
+
+```shell
+PM > Install-Package Arcus.WebApi.Logging.AzureFunctions
+```
+
+### Usage
+
+To make sure the correlation is added to the HTTP response, following additions have to be made in the `.Configure` methods of the function's startup:
+
+```csharp
+[assembly: FunctionsStartup(typeof(Startup))]
+
+namespace MyHttpAzureFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.AddHttpCorrelation();
+        }
+    }
+}
+```
+
+When this addition is added, you can use the `HttpCorrelationService` inside your function to call the correlation functionality and use the `ICorrelationInfoAccessor` (like before) to have access to the `CorrelationInfo` of the HTTP request.
+
+```csharp
+public class MyHttpFunction
+{
+    private readonly HttpCorrelationService _correlationService;
+    private readonly ICorrelationInfoAccessor _correlationAccessor;
+
+    public MyHttpFunction(HttpCorrelationService correlationService, ICorrelationInfoAccessor correlationAccessor)
```

Comment:
Moved the HTTP correlation functionality to a service, used in both Azure Functions and API projects. But not sure if we should merge this with the correlation accessor in the Azure Functions variant...
There are also projects that create some kind of middleware alternative for Azure Functions but that's maybe an idea if we have additional functionality we want to move to Azure Functions.

Reply:
What if we just call it `HttpCorrelation`?

In terms of the alternative, do you have a link to an example?

Reply:
> In terms of the alternative, do you have a link to an example?

https://github.com/umamimolecule/azure-functions-http-middleware#introduction

Reply:
Suggested renaming is good.